gen/builders/builder_message.go

```diff
@@ -146,8 +151,10 @@ func (b *MessageVectorBuilder) Finish(w io.Writer) {
 
 	msgs := b.Messages.All()
 	traces := make([]types.ExecutionTrace, 0, len(msgs))
-	for _, msgs := range msgs {
-		traces = append(traces, msgs.Result.ExecutionTrace)
+	for _, msg := range msgs {
+		if msg.Result != nil {
+			traces = append(traces, msg.Result.ExecutionTrace)
+		}
```

Comment:
I wonder if we want to make the traces a slice of pointers `[]*types.ExecutionTrace`, and leave a nil element for each message that doesn't have a result?